src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public Ball(String color, int number) {
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public int getNumber() {
+        return number;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public void setNumber(int number) {
+        this.number = number;
+    }
+
+    @Override
+    public String toString() {
+        return "Ball{"
+                +
+                "color='" + color + '\''
+                +
+                ", number=" + number
+                +
+                '}';
+    }
```

Comment:
you may change default toString() generation layout in IntellijIdea to comply with checkstyle and not to contain unnecessary lines
```suggestion
    @Override
    public String toString() {
        return "Ball{"
                +  "color='" + color + '\''
                +  ", number=" + number
                +  '}';
    }
```